client/my-sites/site-settings/custom-content-types/index.jsx

```diff
@@ -140,11 +141,17 @@ class CustomContentTypes extends Component {
 		const { translate } = this.props;
 		const fieldLabel = translate( 'Testimonials' );
 		const fieldDescription = translate(
-			'Add, organize, and display {{link}}testimonials{{/link}}. If your theme doesn’t support testimonials yet, ' +
+			'Add, organize, and display {{link /}}. If your theme doesn’t support testimonials yet, ' +
 				'you can display them using the shortcode [testimonials].',
 			{
 				components: {
-					link: <a href={ localizeUrl( 'https://wordpress.com/support/testimonials/' ) } />,
+					link: (
+						<InlineSupportLink
+							supportLink={ localizeUrl( 'https://wordpress.com/support/testimonials/' ) }
+							supportPostId={ 97757 }
+							text={ translate( 'testimonials' ) }
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **21** times:
`translate( 'Testimonials', { context: 'Block pattern category'} )` *ES Score: 11*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/custom-content-support-article-dialogs)